tools/frr

```diff
@@ -337,9 +356,9 @@ stop_prio()
         fi
 
         wanted_prio=$1
-        daemon_list=${daemon:-$DAEMONS}
+        daemon_list=$(echo ${daemon:-$DAEMONS}| xargs -n1 | sort -g | xargs)
```

Comment:
What?  why are we sorting?

Reply:
Again this was done for presentation purposes. When running things all at once outputs were quite unpredictable.